lab-taylor/app/html/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
Again, your code looks great!
Found a few things in the form that don't make sense though.